admin/static/coffee/modals.coffee

```diff
@@ -92,7 +92,7 @@ class DeleteDatabaseModal extends ui_modals.AbstractModal
 
 
     on_submit: =>
-        if @$('.verification_name').val() isnt @database_to_delete.get('name')
+        if @$('.verification_name').val().replace(/^\s+|\s+$/g,'') isnt @database_to_delete.get('name')
```

Comment:
I would generally agree, but I wasn't sure which browser versions should be supported. IE 8 for instance, doesn't support String.prototype.trim. See: https://kangax.github.io/compat-table/es5/#String.prototype.trim
If IE8 support isn't important, I could change it to `.trim()`.